{{cookiecutter.project_name}}/clowdapp.yaml

```
@@ -30,6 +30,16 @@ objects:
           requests:
             cpu: ${CPU_REQUEST_SERVICE}
             memory: ${MEMORY_REQUEST_SERVICE}
+        livenessProbe:
+          httpGet:
+            path: /livez
+            port: ${WEB_PORT}
+          initialDelaySeconds: 3
+          periodSeconds: 3
+        readinessProbe:
+          httpGet:
+            path: /readyz
+            port: ${WEB_PORT}
```

Use the `${{VALUE}}` syntax for setting as a JSON value (not a String).

See https://docs.openshift.com/container-platform/4.13/openshift_images/using-templates.html#templates-writing-parameters_using-templates

```suggestion
            port: ${{WEB_PORT}}
          initialDelaySeconds: 3
          periodSeconds: 3
        readinessProbe:
          httpGet:
            path: /readyz
            port: ${{WEB_PORT}}
```